day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-cert-maintenance.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-troubleshooting-cert-maintenance"]
+= Certificate maintenance
+include::_attributes/common-attributes.adoc[]
+:context: telco-troubleshooting-cert-maintenance
+
+toc::[]
+
+Certificate maintenance is required for continuous cluster authentication.
+As a cluster administrator, you must manually renew certain certificates, while others are automatically renewed by the cluster.
```

Comment:
`certain` would be ambiguous to the user. It is advised to specify the conditions. If that's not required, then this line can be skipped. The following links will talk more about it.